packages/expo/src/web/Performance/Systrace.ts

```diff
@@ -0,0 +1,196 @@
+
```

Comment:
Do we need this whole module? What functions are called and can we get away with stubbing them minimally?